inspect.libsonnet

```diff
@@ -143,4 +141,67 @@ local d = import 'doc-util/main.libsonnet';
          }
 
     else '~[ %s ]' % std.join(' , ', [std.toString(input1), std.toString(input2)]),
+
+  '#filterObjects':: d.fn(
+    |||
+      `filterObjects` walks a JSON tree returning all matching objects in an array.
+
+      The `object` argument can either be an object or an array, other types will be
+      ignored.
+    |||,
+    args=[
+      d.arg('object', d.T.any),
+      d.arg('keyF', d.T.func),
+      d.arg('valueF', d.T.func, default='function(obj) [obj]'),
```

Comment:
Can you document these a bit more? Maybe I'm a bit dense but I'm not really sure about the difference